src/database/queries/sessions.js

```diff
@@ -66,26 +67,37 @@ exports.findById = async (id, tenantCode) => {
 
 exports.findSessionWithAttendee = async (sessionId, userId, tenantCode) => {
 	try {
-		// Optimized: Single query with JOIN to get session and attendee data together
-		const query = `
-			SELECT 
-				s.*,
-				sa.id as attendee_id,
-				sa.type as enrolled_type,
-				sa.meeting_info as attendee_meeting_info,
-				sa.joined_at,
-				sa.mentee_id
-			FROM sessions s
-			LEFT JOIN session_attendees sa ON s.id = sa.session_id AND sa.mentee_id = :userId AND sa.tenant_code = :tenantCode
-			WHERE s.id = :sessionId AND s.tenant_code = :tenantCode
-		`
-
-		const result = await Sequelize.query(query, {
-			replacements: { sessionId, userId, tenantCode },
-			type: QueryTypes.SELECT,
+		const result = await Session.findOne({
+			where: {
+				id: sessionId,
+				tenant_code: tenantCode,
+			},
+			include: [
+				{
+					model: SessionAttendee,
+					as: 'attendees', // correct alias
+					required: false,
+					where: {
+						mentee_id: userId,
+						tenant_code: tenantCode,
+					},
+					attributes: ['id', 'type', 'meeting_info', 'joined_at', 'mentee_id'],
+				},
+			],
 		})
 
-		return result.length > 0 ? result[0] : null
+		const attendee = result?.attendees?.[0]
+
+		return result
+			? {
+					...result.get(),
+					attendee_id: attendee?.id || null,
+					enrolled_type: attendee?.type || null,
+					attendee_meeting_info: attendee?.meeting_info || null,
+					joined_at: attendee?.joined_at || null,
+					mentee_id: attendee?.mentee_id || null,
```

Comment:
Do we really need to flatten the response here? Wouldn’t a nested attendee work just fine?
If this ends up causing a lot of downstream changes, could we move the flattening logic into a DTO instead?